src/main/resources/static/index.html

```diff
@@ -1,87 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-    <!-- Importing Stylesheets -->
-    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons" rel="stylesheet">
-    <link href="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.min.css" rel="stylesheet">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
-    <title>Project Caerus</title>
-</head>
-
-<body>
-    <div id="app">
-        <v-app>
-            <v-content>
-                <v-navigation-drawer
-                        v-model="drawer"
-                        absolute
-                        temporary
-                        color="primary"
-                >
-                    <v-list>
-                        <v-list-tile
-                                v-for="item in items"
-                                :key="item.title"
-                                @click="changePage(item.page)"
-                        >
-                            <v-list-tile-action>
-                                <v-icon>{{ item.icon }}</v-icon>
-                            </v-list-tile-action>
-
-                            <v-list-tile-content>
-                                <v-list-tile-title>{{ item.title }}</v-list-tile-title>
-                            </v-list-tile-content>
-                        </v-list-tile>
-                    </v-list>
-                </v-navigation-drawer>
-
-                <v-toolbar dark color="primary">
-                    <v-toolbar-side-icon @click="drawer = !drawer"></v-toolbar-side-icon>
-                    <v-toolbar-title class="white--text">Project Caerus</v-toolbar-title>
-                </v-toolbar>
-
-                <v-btn @click="showLogin">Toggle showing login page</v-btn>
-                <component v-if=show is="login"></component>
-                <component v-if="page === 'home'" is="home"></component>
```

Comment:
So while this would work it'd be a lot simpler to do <component :is="page"></component>
The ':' infront of is allows for js to be computed instead of just a string